public/css/style.css

```diff
@@ -1,3 +1,5 @@
+body { padding-top: 70px; }
```

Comment:
@daria-sieroshtan 
Styling 'body` is considered to be a bad practise
Better style main container. This is the container purpose